### DISCUSSION TITLE
IGMP proxy issue ( default)

### DISCUSSION THREAD

**Discussion by starpc** [fabianishere/udm-iptv#243] [Q&A]
### What is your ISP? (if relevant)

KPN

### Diagnostic Information

root@UDM-SE-Archipel:~# udm-iptv-diag
Please share the following output with the developers:
=== Configuration ===
WAN Interface: eth8
WAN VLAN: 4 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 213.75.0.0/16  217.166.0.0/16
LAN Interfaces: br0
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
61: iptv@eth8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    inet 10.254.97.154/21 brd 10.254.103.255 scope global iptv
       valid_lft forever preferred_lft forever
10.254.96.0/21 proto kernel scope link src 10.254.97.154 
213.75.112.0/21 via 10.254.96.1 metric 261 
=== Service Logs ===
Mar 11 21:29:20 UDM-SE-Archipel udm-iptvd[507659]: RECV V2 member report   from 192.168.10.252  to 239.255.3.22
Mar 11 21:29:20 UDM-SE-Archipel udm-iptvd[507659]: Updated route entry for 239.255.3.22 on VIF #0
Mar 11 21:29:20 UDM-SE-Archipel udm-iptvd[507659]: RECV V2 member report   from 192.168.10.251  to 224.0.0.22
Mar 11 21:29:20 UDM-SE-Archipel udm-iptvd[507659]: The IGMP message was from myself. Ignoring.
Mar 11 21:29:21 UDM-SE-Archipel udm-iptvd[507659]: The IGMP message was local multicast. Ignoring.
Mar 11 21:29:21 UDM-SE-Archipel udm-iptvd[507659]: RECV V2 member report   from 192.168.10.241  to 239.254.127.63
Mar 11 21:29:21 UDM-SE-Archipel udm-iptvd[507659]: Updated route entry for 239.254.127.63 on VIF #0
Mar 11 21:29:21 UDM-SE-Archipel udm-iptvd[507659]: RECV V2 member report   from 192.168.10.120  to 239.255.188.58
Mar 11 21:29:21 UDM-SE-Archipel udm-iptvd[507659]: Updated route entry for 239.255.188.58 on VIF #0
Mar 11 21:29:21 UDM-SE-Archipel udm-iptvd[507659]: The IGMP message was local multicast. Ignoring.

### Describe the Bug

i have updated the unifi OS on my UDM SE to v3.0.18
before i had no issue with the TV. but the tv stops and starts. it looks like its related to the IGMP proxy (default) option. 
 
 i played around with the other option "improxy > IGMP v3" and that one looks like to work with no error's 
 
perhaps the update messed up something. i did re-install the TV plugin. 

also, the configuration is now set up for WAN RJ45. when i started using the Plugin. i was a bit confused. 

the install script calls for 
WAN 1 (RJ45) 
WAN 2 (SFP+) 

it's not really clear what port is what. 
Currently i have my WAN via Eht9. the script is configured for eth8. 

i found out by looking in note pad at the files to understand that 
wan 1 is port 8
wan 2 is port 9 

case "$board" in
            UDMPRO|UDMPROSE|UDMSE)
                db_subst udm-iptv/wan-port choices "WAN 1 (RJ45), WAN 2 (SFP+)"
                db_subst udm-iptv/wan-port choices_c "eth8, eth9"
                db_input high udm-iptv/wan-port || true
                
                
  i have to report that the WAN2 option for eth9 is not working. for port 8 the config is working but i dont understand why. 
  
  please help.   
               

### Expected Behavior

look for my sugestions above please thx so much

**Comment by fabianishere** [comment]
If improxy is working for you, please use that. It will eventually replace igmpproxy as default.

Also, Port 9 (RJ45 WAN port) on the UDM is called eth8 in Linux and Port 10 (SFP+ WAN port) is called eth9 in Linux, since Linux counts from 0, whereas the UDM labels the ports from 1.

**Comment by FThuijs** [comment]
Just to inform you:
I had the same problem (not working anymore) when I updated my UDM-PRO to version 2.5.17. I tried everything but what finally did the trick was uninstalling udm-iptv and the installing it again. I then could use igmpproxy.
Did you also tried that?
